File: TenMinsJava/day27.java
// Create a function that will subtract y from x, but only until zero
package TenMinsJava;

public class day27 {
    public static void sub(int x, int y) {
        boolean t = true;
        int sum;
        while (t) {
            if (x - y <= 0) {
                System.out.println(0);
                t = false;
            }
            else {
                sum = x - y;
                x = sum;
                System.out.println(sum);
            }
        }
    }
    public static void main(String[] args) {
        sub(100, 10);
    }
}

File: TenMinsPy/day27.py
# Create a function that will subtract y from x, but only until zero
class day27:
    def sub(x, y):
        t = True
        while t:
            if x - y <= 0:
                print('0')
                t = False
            else:
                print(x - y)
                sum = x - y
                x = sum
    sub(100, 10)

day27()